corehq/apps/api/tests/form_resources.py

```diff
@@ -67,6 +70,14 @@ def test_fetching_xform_cases(self):
         self.assertEqual(len(cases), 1)
         self.assertEqual(cases[0]['id'], case_id)
 
+    def test_filter_forms_by_cases_modified(self):
+        form, case_id = self._submit_case_update_form()
+        response = self._assert_auth_get_resource(f"{self.list_endpoint}?case_id={case_id}")
```

Comment:
what happens if you provide multiple case IDs: `?case_id=A&case_id=B`?

Reply:
It'll just use the last one (same as with things like `xmlns` or `case_type`).  Though I checked, and `es_query_from_get_params` does actually have access to the raw `QueryDict`, which contains multiple keys, so this could be refactored or special-cased to use `request.GET.getlist()`.  Is that what you're thinking?

Reply:
no need to change it now, I just wanted to check.